dockyard/controllers/v1/__init__.py

```diff
@@ -0,0 +1,17 @@
+from dockyard.controllers.v1 import information
+from dockyard.controllers.v1 import container
+from dockyard.controllers.v1 import image
```

Comment:
i think, importing can be done in such a way that it supports all the APIs

Therefor importing like 
from dockyard.controllers import v1

will make sure that different versions APIs can be added here.